include/gzip.hpp

```diff
@@ -3,7 +3,8 @@
  * gzip implements a standard namespace with a few available functions.
  */
 
-#include "gzip/compress.hpp"
-#include "gzip/decompress.hpp"
-#include "gzip/utils.hpp"
-#include "gzip/version.hpp"
\ No newline at end of file
+#include <gzip/compress.hpp>
+#include <gzip/config.hpp>
+#include <gzip/decompress.hpp>
+#include <gzip/utils.hpp>
+#include <gzip/version.hpp>
```

Comment:
^^ this is unrelated to fixing casts. Rather it is best practice to always include files the same way with `<>` consistently.